x-pack/solutions/security/plugins/elastic_assistant/server/lib/attack_discovery/schedules/register_schedule/executor.ts

```diff
@@ -20,17 +48,84 @@ export const attackDiscoveryScheduleExecutor = async ({
   options,
   logger,
 }: AttackDiscoveryScheduleExecutorParams) => {
-  const { services } = options;
-  const { alertsClient } = services;
+  const { params, services, spaceId } = options;
```

Comment:
@pmuellr @mikecote JFYI, this is the implementation of the Attack Discovery Schedule rule executor which was introduced without implementation earlier in this PR https://github.com/elastic/kibana/pull/216656/files#diff-8c25b58c667ec0edcc06c6dc1646cfbf16fdea684432299310648b728c1f5136R29